p2p/simulations/network.go

```diff
@@ -388,7 +399,7 @@ func (self *Network) Stop(id discover.NodeID) error {
 	node.Up = false
 	log.Info(fmt.Sprintf("stop node %v: %v", id, node.Up))
 
-	self.events.Send(ControlEvent(node))
+	self.events.Send(NewEvent(node))
```

Comment:
@lmars I didn't figure out what is wrong, but it definitely only worked after this. What do you propose then?